docs/index.md

```diff
@@ -166,4 +166,66 @@ module operator_roles {
     rh_oidc_provider_url = ocm_cluster_rosa_classic.rosa_sts_cluster.sts.oidc_endpoint_url
     operator_roles_properties = data.ocm_rosa_operator_roles.operator_roles.operator_iam_roles
 }
+```
+
+## Advanced Usages
```

Comment:
Please do not edit the index.md manually - I tried to talk about it in the last sync meeting, after running make generate, the generate tool replaced that file with an auto-generated... 
we need to find a different way to edit that file